tests/wpt/meta/css/css-grid/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-002.html.ini

```diff
@@ -1,2 +1,3 @@
 [grid-self-baseline-changes-grid-area-size-002.html]
-  expected: FAIL
+  expected:
+    if os == "linux": FAIL
```

Comment:
Why is this os-dependent?

Reply:
It's believe it's related to text rendering. I agree that it shouldn't be os-dependent, but there are a few layout related tests that are for some reason. IIRC, some of them also only fail when run as a large batch of tests or in debug mode, suggesting there's some kind of race condition occurring.

Reply:
But CI only runs tests in linux. Are you changing this because this PR fixes the test for you on a non-linux system? Or how do you know it's os-dependent?